keyboards/foxlab/key65/hotswap/readme.md

```diff
@@ -1,6 +1,6 @@
 # Foxlab Key 65 Hotswap
 
-![Key 65](https://i.imgur.com/28Qck8ul.jpg)
+![Key 65](https://i.imgur.com/8HpYStP.jpg)
```

Comment:
```suggestion
![Key 65](https://i.imgur.com/28Qck8ul.jpg)
```

Reply:
My bad. Didn't mean to change this. Thanks